kdbx/src/main/java/org/linguafranca/pwdb/kdbx/KdbxKeyFile.java

```diff
@@ -21,13 +21,20 @@
 import org.linguafranca.pwdb.security.Encryption;
 import org.w3c.dom.Document;
 
+import com.google.common.io.ByteStreams;
+
 import javax.xml.parsers.DocumentBuilder;
 import javax.xml.parsers.DocumentBuilderFactory;
 import javax.xml.xpath.XPath;
 import javax.xml.xpath.XPathConstants;
 import javax.xml.xpath.XPathFactory;
+
+import java.io.ByteArrayInputStream;
 import java.io.InputStream;
 import java.security.MessageDigest;
+import java.util.Arrays;
+
+
 
 /**
  * Class has a static method to load a key from a KDBX XML Key File
```

Comment:
Comment needs updating to reflect that you have implemented for all types of Key file. Would be worth documenting what they are

Reply:
You're right! Tonight, I will update the comment!